apps/condo/domains/ticket/components/IncidentHints.tsx

```diff
@@ -138,42 +137,26 @@ const WORK_FINISHED_IN_LAST_DAYS = 7
 export const IncidentHints: React.FC<IncidentHintsProps> = (props) => {
     const { propertyId, classifier, organizationId, colProps } = props
 
-    const [allIncidents, setAllIncidents] = useState<IIncident[]>([])
-    const [incidentsToShow, setIncidentsToShow] = useState<IIncident[]>([])
+    const [allIncidents, setAllIncidents] = useState<GetIncidentsQuery['incidents']>([])
+    const [incidentsToShow, setIncidentsToShow] = useState<GetIncidentsQuery['incidents']>([])
 
-    const { refetch: refetchIncidentProperties } = useGetIncidentPropertiesQuery({ skip: true })
-    const { refetch: refetchIncidents } = useGetIncidentsQuery({ skip: true })
-    const { refetch: refetchIncidentClassifierIncidents } = useGetIncidentClassifierIncidentQuery({ skip: true })
+    const [getIncidentProperties] = useGetIncidentPropertiesByPropertyIdAndRecentIncidentsLazyQuery()
+    const [getIncidents] = useGetIncidentsLazyQuery()
+    const [getIncidentClassifierIncidents] = useGetIncidentClassifierIncidentLazyQuery()
 
-    const categoryId = useMemo(() => get(classifier, 'category.id', null), [classifier]) as string | null
-    const problemId = useMemo(() => get(classifier, 'problem.id', null), [classifier]) as string | null
+    const categoryId = useMemo(() => classifier?.category?.id || null, [classifier]) as string | null
+    const problemId = useMemo(() => classifier?.problem?.id || null, [classifier]) as string | null
 
     const fetchIncidentProperties = useCallback(async (propertyId: string, organizationId: string) => {
-        const res = await refetchIncidentProperties({
-            where: {
-                property: {
-                    id: propertyId,
-                    deletedAt: null,
-                },
-                incident: {
-                    organization: { id: organizationId },
-                    OR: [
-                        {
-                            AND: [{ status: IncidentStatusType.Actual }],
-                        },
-                        {
-                            AND: [{
-                                status: IncidentStatusType.NotActual,
-                                workFinish_gte: dayjs().subtract(WORK_FINISHED_IN_LAST_DAYS, 'days').toISOString(),
-                            }],
-                        },
-                    ],
-                },
-                deletedAt: null,
+        const { data:  incidentPropertiesData } = await getIncidentProperties({
+            variables: {
+                propertyId,
+                organizationId,
+                workFinish_gte: dayjs().subtract(WORK_FINISHED_IN_LAST_DAYS, 'days').toISOString(),
```

Comment:
Let's do something with this query so that it's obvious what we're getting as a result

Reply:
Ahh, that's a new request
Let's make this function name as obvious as the hook name